reactapp/src/components/ListElement/ListElement.js

```diff
@@ -1,12 +1,31 @@
+import '../ListElement/styles.scss';
+import { useState, useEffect } from "react";
 
 export const ListElement = ({children}) => {
 
-  const handleClick = () => {
-    console.log('element listy:', children);
+  const [ pageTitle, setPageTitle ] = useState('');
+  const [ changeClass, setChangeClass ] = useState('');
+  // const [ removeClass, setRemoveClass ] = useState('mark-as-read');
+
+
+  const handleOnClick = (event) => {
+    setPageTitle(event.target.innerText);
+    setChangeClass('mark-as-read');
```

Comment:
zamiast uzywac stanu to ustawienia konkretnej klasy, lepiej traktowac to jako boolean, czyli np isActive, setIsActive i potem na podstawie tej wartosci w komponencie ustawiac klase w taki sposob:

className={ isActive ? 'className' : '' }

czyli jak isActive zwroci true to wstawi taka klase jaka chcesz a jak nie to wstawic pusty string czyli defakto usunie te klase